cypress/fixtures/systemRecommendations.json

```diff
@@ -0,0 +1,89 @@
+      "summary": "System is not able to get the latest recommendations and may miss bug fixes when the Insights Client Core egg file is outdated.\n",
+      "generic": "The Insights Client Core egg file is outdated, which results in not being able to get the latest recommendations and may miss bug fixes for older recommendations.\n",
+      "reason": "The Insights Core egg file used on this host is outdated:\n\n* The current version: {{=pydata.cur}}\n* The latest version: {{=pydata.rel}}\n",
+      "more_info": "For more information about how to disable/enable automatic rule updates for Red Hat Insights, refer to:\n- [CHANGING RED HAT INSIGHTS AUTOMATIC RULE UPDATES](https://access.redhat.com/documentation/en-us/red_hat_insights/2020-10/html/client_configuration_guide_for_red_hat_insights/changing-automatic-rule-update)\n",
+      "resolution_set": [
+        {
+          "system_type": 105,
+          "resolution": "{{?pydata.auto_update}}\nRed Hat recommends that you remove/comment the **auto_update=False** line in the `/etc/insights-client/insights-client.conf` or modify it to **auto_update=True** and then run the `insights-client` command again.\n{{??}}\nRed Hat recommends that you update the Red Hat Insights Client Core egg file manually:\n1. Update the egg to the latest manually:\n  ~~~\n  # curl https://api.access.redhat.com/r/insights/v1/static/core/insights-core.egg > /var/lib/insights/last_stable.egg\n  # curl https://api.access.redhat.com/r/insights/v1/static/core/insights-core.egg.asc > /var/lib/insights/last_stable.egg.asc\n  ~~~\n1. Check the version:\n  ~~~\n  # insights-client --version\n  ~~~\n{{?}}\n",
+          "resolution_risk": {
+            "name": "Update Insights Configuration",
+            "risk": 2
+          },
+          "has_playbook": true
+        }
+      ],
+      "total_risk": 2
+    },
+    "details": {
+      "cur": "3.0.246-1",
+      "rel": "3.0.301",
+      "rhel": "8.6",
+      "type": "rule",
+      "error_key": "INSIGHTS_CORE_EGG_NOT_UP2DATE"
+    },
+    "resolution": {
+      "system_type": 105,
+      "resolution": "{{?pydata.auto_update}}\nRed Hat recommends that you remove/comment the **auto_update=False** line in the `/etc/insights-client/insights-client.conf` or modify it to **auto_update=True** and then run the `insights-client` command again.\n{{??}}\nRed Hat recommends that you update the Red Hat Insights Client Core egg file manually:\n1. Update the egg to the latest manually:\n  ~~~\n  # curl https://api.access.redhat.com/r/insights/v1/static/core/insights-core.egg > /var/lib/insights/last_stable.egg\n  # curl https://api.access.redhat.com/r/insights/v1/static/core/insights-core.egg.asc > /var/lib/insights/last_stable.egg.asc\n  ~~~\n1. Check the version:\n  ~~~\n  # insights-client --version\n  ~~~\n{{?}}\n",
+      "resolution_risk": { "name": "Update Insights Configuration", "risk": 2 },
+      "has_playbook": true
+    },
+    "impacted_date": "2022-11-02T10:08:50.888206Z"
```

Comment:
Can we write one more test to check that the table translates and renders the timestamp correctly? You can just choose one row for this check.

Reply:
Let me know if this is what you expected.. Other, simpler solution would be to only check if the timestamp value contains 'ago' and maybe that would be sufficient as well